qc_utils.py

```diff
@@ -345,45 +350,66 @@ def gcv_calculate_binmax(indata: np.array, binmin: float, binwidth: float) -> fl
     return binmax  #gcv_calculate_binmax
     
 
-
-def gcv_central_section(full_hist: np.array) -> tuple[int, int]:
+#*********************************************
+def gcv_zeros_in_central_section(histogram: np.array, inner_n: int) -> tuple[int, int]:
     """
     A routine to determine if, for a distribution with multiple peaks, the central
     section is sufficiently big.
```

Comment:
What meteorological issue is this trying to find? Something unusual about the distribution but can you give more info here in the docstring? Why do we believe this to be an issue?

Reply:
Hopefully I've improved the docstring.  This is a helper routine for the get_critical_values() function to count how many zero-valued bins there are in the `inner_n` bins, as if many are zero, then the x^-1 line fitting didn't work well on some occaisions.  However, your comment did highlight that the typing didn't reflect the current format of this function, so I've updated that.